internal/server/data/data.go

```diff
@@ -116,11 +120,43 @@ func get[T models.Modelable](db *gorm.DB, selectors ...SelectorFunc) (*T, error)
 	return result, nil
 }
 
+func getOrg(db *gorm.DB) *models.Organization {
+	org, ok := db.Statement.Context.Value("org").(*models.Organization)
+	if !ok {
+		// TODO(orgs): panic("no org") when not set
+		return &models.Organization{}
+	}
+	return org
+}
+
+func setOrg(db *gorm.DB, model any) {
+	member, ok := model.(orgMember)
+	if !ok {
+		return
+	}
```

Comment:
When we are ready to, we can make `SetOrganizationID` a required method, and implement no-op implementations on the other tables. That gives us the same safeguards as putting it in the base model, without adding unused organization_id fields.